test/extensions/coin-machine.js

```diff
@@ -404,7 +404,7 @@ contract("Coin Machine", (accounts) => {
       const periodLength = await coinMachine.getPeriodLength();
       const maxPerPeriod = await coinMachine.getMaxPerPeriod();
       const windowSize = await coinMachine.getWindowSize();
-      const alphaAsWad = new BN(2).mul(WAD).divRound(windowSize.addn(1));
+      const alphaAsWad = new BN(2).mul(WAD).div(windowSize.addn(1));
```

Comment:
Can you mention in the PR description why we don't need `divRound` anymore?

Reply:
Is this not also covered in point 2? I can make it more explicit if you like.

Reply:
Mm it's alright.